backend/app/models/user.py

```diff
@@ -10,10 +10,19 @@ class User(db.Model):
     id = db.Column(db.Integer, primary_key=True, nullable=False)
     first_name = db.Column(db.String, nullable=False)
     last_name = db.Column(db.String, nullable=False)
-    auth_id = db.Column(db.String, nullable=False)
+    auth_id = db.Column(db.String, nullable=True)
     role = db.Column(
         db.Enum("Admin", "Regular Staff", "Relief Staff", name="roles"), nullable=False
```

Comment:
Likewise with the change made on the FE, I think it'd be beneficial to export a common enum from somewhere (maybe create a constants file in the `utilities` dir) and use it here, just so we can minimize the number of areas that need to be changed everytime we change the values for the possible roles. Something like this could be implemented (shouldn't be any compilation issues)

```suggestion
# Somewhere above in the file / in the /utilities directory
  class RoleOptions(enum.Enum):
      ADMIN = 'Admin'
      RELIEF_STAFF = 'Relief Staff'
      REGULAR_STAFF = 'Regular Staff'


        role = db.Column(
        db.Enum(RoleOptions, values_callable=lambda x: [str(role.value) for role in RoleOptions], name="roles"), nullable=False
    )
```

Reply:
hmm @kevin-pierce i think it already has an enum of type roles created in the db, so anytime we use / reference it in the table it doesn't actually create the new enum type of roles

Reply:
or at least that's the way it is in the db, and yeah i agree w safwaan that it could cause a lot of conflicts in the migration files so may not be the most worth what do you think?

Reply:
Yeah fair enough - that wasn't something I personally tested when I suggested that change, so I think it's fine to leave as is if it could cause trouble with the migration files